cluster/store/service.go

```diff
@@ -298,40 +299,40 @@ func (s *Service) WaitUntilDBRestored(ctx context.Context, period time.Duration)
 
 // QueryReadOnlyClass will verify that class is non empty and then build a Query that will be directed to the leader to
 // ensure we will read the class with strong consistency
-func (s *Service) QueryReadOnlyClass(class string) (*models.Class, uint64, error) {
-	if class == "" {
-		return &models.Class{}, 0, fmt.Errorf("empty class name: %w", errBadRequest)
+func (s *Service) QueryReadOnlyClasses(classes ...string) (map[string]classcache.VersionedClass, error) {
+	if len(classes) == 0 {
```

Comment:
question(non-blocking): should we also check that all classes passed as parameter are non empty string ? Or do we assume this is an error that will bubble up later down the line that a class was an empty string ?

Reply:
yes, also remove any duplication